packages/safe-tools-client/app/components/safe-info/index.gts

```diff
@@ -27,6 +33,15 @@ interface Signature {
 }
 
 export default class SafeInfo extends Component<Signature> {
+  @tracked isShowingCopiedConfirmation = 0;
+
+  @action flashCopiedConfirmation() {
```

Comment:
Consider converting to a concurrency task to get better async lifecycle management. i.e. shouldn't try to set execute `this.isShowingCopiedConfirmation = 0` after delay if component has been destroyed.